src/assets/image/icons/index.ts

```diff
@@ -59,6 +59,7 @@ export { default as ReceiptIcon } from "@assets/image/icons/Receipt.svg?react";
 export { default as RedditIntroIcon } from "@assets/image/icons/Reddit.svg?react";
 export { default as RocketIcon } from "@assets/image/icons/Rocket.svg?react";
 export { default as RotateIcon } from "@assets/image/icons/Rotate.svg?react";
+export { default as RotateRightIcon } from "@assets/image/icons/RotateRight.svg?react";
```

Comment:
Please add the exact link to the icon page in FontAwesome.

Reply:
added